samples/ra-vpn/saml-multi-region/saml-cert.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   namespace: sfcn-system
 stringData:
   trustpoint: samlpkcs
+  format: pkcs12
```

Comment:
Is this correct? We aren't importing this cert but also this isn't pkcs12 formatted right?

Reply:
IDK, it says **samlpkcs** so I thought it's pkcs12.
I'll check

Reply:
@jaredhancock31 , okay, addressed, and also made `sfcn-redis` consistent with other examples

Reply:
@jaredhancock31 , that `sfcn-redis` is created by the helm chart. Reverted that change 😞